### PR TITLE
DE-228 feat: add subgraph to track MetaAggregationRouter Fee

### PR DESCRIPTION
**Pull request by lehainam-dev**

### CODE REVIEW

File: meta-aggregation-fee/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+type ClientData @entity(immutable: true) {
+  id: ID!
+  data: String!
+
+  tx: Bytes!
+  blockNumber: BigInt!
+  time: BigInt!
+}
+
+type RecipientFee @entity(immutable: true) {
```
Comment (halink0803) [inline meta-aggregation-fee/schema.graphql:10]: Can you please explain the immutable logic?

**Review by halink0803 — APPROVED** [review]
I hope you could add unit test if possible.

File: meta-aggregation-fee/schema.graphql (hunk 1 above)
Comment (lehainam-dev) [reply meta-aggregation-fee/schema.graphql:10]: @halink0803 So for a short explanation, if the entity is never changing after the block it created, it should be marked as immutable.

For a long answer, you can visit this section in subgraph document: https://thegraph.com/docs/en/developing/creating-a-subgraph/#defining-entities

Comment (halink0803) [reply meta-aggregation-fee/schema.graphql:10]: Yes, I know the immutable definition, but the `recipient` fee is only create one time and never be changed then. Then it is okay 👍

**Pull request merged**